requirements.txt

```diff
@@ -9,3 +9,7 @@ nose-timer
 # necessary for accessing SecretStorage keyring (system wide Gnome keyring)
 # and nice to have under tox for that
 # dbus-python
+
+# -- required for building docs
+sphinx
+sphinx_rtd_theme
```

Comment:
it might be better to create a new `docs` section (along with meta etc) within setup.py and place it there, then it would also be used by e.g. pip if you do 'pip install repronim[full]'